controller/src/controller.rs

```diff
@@ -131,35 +135,36 @@ where
 
 	// If so configured, add the foreign API to the same port
 	if owner_api_include_foreign.unwrap_or(false) {
-		info!("Starting HTTP Foreign API on Owner server at {}.", addr);
-		let foreign_api_handler_v2 = ForeignAPIHandlerV2::new(wallet.clone());
+		warn!("Starting HTTP Foreign API on Owner server at {}.", addr);
```

Comment:
why `info`->`warn` ? if we want to see the messages we can set the log level, these don't look like warnings to me

Reply:
Changed this for consistency with how it's done in other places. The default in grin.toml is `warn`, and I think we want to make sure something like that is definitely printed to the console by default.